foundation/foundation-mda/src/main/resources/templates/notifications/notification.deploy.clean.app.target.vm

```diff
@@ -1,6 +1,5 @@
             <plugin>
                 <artifactId>maven-clean-plugin</artifactId>
-                <version>${version.clean.plugin}</version>
```

Comment:
Q: should this be `${version.maven.clean.plugin}` instead of removing it?

Reply:
The version is inherited from the parent pom, so it isn't needed at all here.